File: qlty-cli/tests/lang/csharp/basic.in/BooleanLogic.cs
public class BooleanLogic
{
    private int foo;
    private int bar;

    private void F0()
    {
        var x = foo - bar + 1;
    }
}

public class BooleanLogic1
{
    private bool foo;
    private bool bar;
    private bool baz;
    private bool qux;
    private bool zoo;

    private void F1()
    {
        if (foo && bar && baz && qux && zoo)
        {
            return;
        }
    }
}

File: qlty-cli/tests/lang/swift/basic.in/BooleanLogic.swift
class BooleanLogic {
    var foo: Int = 0
    var bar: Int = 0
    
    func f0() {
        let x = foo - bar + 1
    }
}

class BooleanLogic1 {
    var foo: Bool = false
    var bar: Bool = false
    var baz: Bool = false
    var qux: Bool = false
    var zoo: Bool = false
    
    func f1() {
        if foo && bar && baz && qux && zoo {
            return
        }
    }
}